CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Prowide ISO 20022 - CHANGELOG
 
+
+#### 9.4.5-SNAPSHOT May 2023
+  * (PW-1875) Added support for the BusinessApplicationHeaderV01 to always have CreDt with Zulu timezone.
```

Comment:
(PW-1875) Changed the BusinessApplicationHeaderV01 marshaller to always use Zulu timezone with "Z" indicator